lib/boot.js

```diff
@@ -1,3 +1,19 @@
+const foundElectron = window.process && window.process.type;
+
+if (!foundElectron) {
+  if (localStorage.getItem('appOpened')) {
+    const message = document.createElement('h1');
+    message.innerText =
+      'Simplenote cannot currently be opened simultaneously in more than one tab or window per browser';
+    document.body.appendChild(message);
```

Comment:
this is fine though we are free to use React since it will have been delivered in this bundle already